docs/fleet/cli.md

```diff
@@ -702,19 +702,103 @@ You can use the `module build start` or `module build local` commands to build y
 - Use `build start` to build or compile your module on a cloud build host that might offer additional platform support than you have access to locally.
 - Use `build local` to quickly test that your module builds or compiles as expected on your local hardware.
 
-To configure your module's build steps, add a `build` object to your [`meta.json` file](#the-metajson-file), including the following:
+To configure your module's build steps, add a `build` object to your [`meta.json` file](#the-metajson-file).
+Developers can either have a single build file for all platforms, or platform specific files:
+
+{{< tabs >}}
+{{% tab name="Single Build File" %}}
 
 ```json {class="line-numbers linkable-line-numbers"}
 "build": {
   "setup": "./setup.sh",                  // optional - command to install your build dependencies
-  "build": "make module.tar.gz",          // command that will build your module
-  "path" : "module.tar.gz",               // optional - path to your built module
+  "build": "sh build.sh",                 // command that will build your module
```

Comment:
would change this to `./build.sh` unless someone objects. doing it this way runs it under the `sh` shell, which is weird because the shebang in 'example build.sh' below uses bash

(sh doesn't support the `source` command, which we use sometimes)